### ISSUE TITLE
Parsing of broken incoming JSON data failed with "Internal Server Error" 

### ISSUE THREAD

**Issue by sw-kosaki** [simpleidserver/SimpleIdServer#79]
Hello again,

we found that when broken JSON data was sent during the Create user request, the parser method dedicated for reading of incoming json data throws an JsonReader exception and respectively the server Response is "Internal Server Error" with Status 500.

For example our **Request body**:
```
{
    "schemas": [
        "urn:ietf:params:scim:schemas:core:2.0:User",
        "urn:ietf:params:scim:schemas:extension:enterprise:2.0:User"
    ],
    "externalId": "KR",
    "userName": "test@bps.com",
    "active": 
    "meta": {
        "resourceType": "User"
    }
}
```

And relevant **Response**:
```
System.NullReferenceException: Object reference not set to an instance of an object.
   at SimpleIdServer.Scim.Api.BaseApiController.InternalAdd(RepresentationParameter jobj) in C:\Users\Konstantin.Rizov\source\repos\user-provisioning-service\src\SimpleIdServer.Scim\Api\BaseApiController.cs:line 236
....
```
   
**Exception** in the code:
    
![Json_Parser_exception](https://user-images.githubusercontent.com/58478592/105731530-2c3dff80-5f38-11eb-8b14-39ff91c9d1c4.PNG)

**Comment by simpleidserver** [comment]
Hello,

The issue should be fixed in the branch "origin/release/1.2.0"

**Comment by sw-kosaki** [comment]
Thank you, the problem was fixed!
